test/test_network_service.py

```diff
@@ -43,15 +44,14 @@ def test_network_properties(self):
             sit.check_repr(self, net)
 
             self.assertIn(
-                net.cidr_block, ['', '10.0.0.0/16'],
+                net.cidr_block, ['', '10.0.0.0/16', '10.128.0.0/9'],
```

Comment:
Can you clarify why the second CIDR block is needed? It doesn't match the one we specified when creating the network?